### PR TITLE
MBS-11304: Add recordings’ first release date to release lookup

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-11304

Release is needed for Picard.

### CODE REVIEW

**Review by yvanzo — APPROVED** [review]
* Checked parity with search results (which just do not return recordings but at top level)
* Checked compatibility with/without active schema 26
* Updated ticket MBS-11304 accordingly

Note: `inc=recording-rels` is not affected.

**Comment by yvanzo** [comment]
@reosarevok: It requires updating https://wiki.musicbrainz.org/MusicBrainz_API though.

**Comment by reosarevok** [comment]
@yvanzo: you mean the examples at https://wiki.musicbrainz.org/MusicBrainz_API/Examples ? I don't see anywhere on the basic page that this would be mentioned :)

**Comment by yvanzo** [comment]
Right, please link to the corresponding wiki diff(s).

File: t/lib/t/MusicBrainz/Server/Controller/WS/2/JSON/LookupRelease.pm [hunk 1]
```diff
@@ -792,6 +796,7 @@ test 'release lookup with discids' => sub {
                                 length => 213106,
                                 disambiguation => "",
                                 video => JSON::false,
+                                "first-release-date" => '2001-07-04',
```
Comment (mwiencek) [inline t/lib/t/MusicBrainz/Server/Controller/WS/2/JSON/LookupRelease.pm:799]:
```suggestion
                                "first-release-date" => '2001-07-04',
```

**Review by mwiencek — CHANGES_REQUESTED** [review]
Mentioned this on IRC, but I'd prefer not adding it to /ws/2/artist/?inc=recordings if Picard doesn't need it. That feature is useless (since it returns a fixed list of 25 recordings by the artist) and supplanted by proper recording browse requests. Returning more information there might signal to people in the patch notes that they should actually be using that feature.

**Comment by reosarevok** [comment]
Dropped it from artists then.

**Comment by reosarevok** [comment]
Ok, removed it from artist then :)

**Review by mwiencek — APPROVED** [review]

**Pull request merged**